server/webdriver/cdc_tests/autopush/download_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tempfile
+
+from server.webdriver.base import WebdriverBaseTest
+from server.webdriver.cdc_tests.base import CDC_AUTOPUSH_URL
+from server.webdriver.shared_tests.download_test import DownloadTestMixin
+
+
+# Class to test the download tool for Custom DC
+class TestDownload(DownloadTestMixin, WebdriverBaseTest):
+  DATACOMMONS_STRING = "Custom Data Commons"
+
+  def get_server_url(self):
```

Comment:
Perhaps create a `CdcAutopushTestBase` (or similarly named) class that provides these common fields and methods and have all CDC tests extend it?

Reply:
Done!